zeppelin-interpreter/pom.xml

```diff
@@ -36,6 +36,14 @@
   <url>http://zeppelin.incubator.apache.org</url>
 
   <dependencies>
+
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>zeppelin-spark-utils</artifactId>
```

Comment:
It is not because of adding new module, but because of zeppelin-interpreter has new module as a dependency. 

Zeppelin want to minimize the dependency version conflict between interpreters. So, if zeppelin-interpreter has zeppelin-spark-utils as a dependency then spark-core and it's transitive dependency will be included in all other interpreter, too.